rehydrate/fargate/main.go

```diff
@@ -17,7 +18,7 @@ const ThresholdSize = int64(100 * 1024 * 1024)
 var awsConfigFactory = awsconfig.NewFactory()
 
 func main() {
-	// The os.Exit calls below make this function untestable. Reason for the os.Exit is so
+	// The os.Exit call below make this function untestable. Reason for the os.Exit is so
```

Comment:
This seems like a reasonable strategy (we have too many failures that on the surface are reported as successes elsewhere). **I would really like to see some ideas on how we can surface these to get our attention.** Part of a SPARC Y7 milestone is to improve this sort of thing -- early notification of significant failures (like this) to our team. My initial thoughts are: (1) DataDog dashboard, (2) Slack notification. Having good logs and tools should make the troubleshooting "quick and easy."